language/move-lang/functional-tests/tests/diem/validator_set/remove_validator.move

```diff
@@ -13,7 +13,8 @@
 // remove_validator cannot be called on a non-validator
 script{
     use 0x1::DiemSystem;
-    fun main(account: &signer) {
+    fun main(account: signer) {
+    let account = &account;
```

Comment:
I wonder what your longer term plan is. Do you plan to eventually refactor these tests and get rid of the let bindings or just leave them there?

Reply:
I'm not sure if I follow, the bindings are fine. Nothing wrong with them. What did you have in mind?

Reply:
Oh I was just wondering if you were planning to clean them up a little bit more. For example, 
```
fun main(account: signer) {
    let account = &account;
    assert(!Roles::can_hold_balance(account), 1);
}
}
```
becomes
```
fun main(account: signer) {
    assert(!Roles::can_hold_balance(&account), 1);
}
}
```
I don't have a strong opinion on this. Just curious what you think

Reply:
Ah, I don't think it is really worth cleaning it up for the sake of doing it. The compiler will actually inline this for me which is fun :) 

(But I know your concern is more about readability)